src/hatti/charting.cljc

```diff
@@ -73,7 +80,8 @@
    metadata of this above value would be:
    {:bins ['1 to 2', '3 to 4', '5 to 6', '7 to 8', '9 to 10']}"
      [answers bins typ]
-     (let [numbers (map (str->int typ) answers)
+     (let [numbers (->> answers
```

Comment:
we've gone back and forth on this, but I don't think this style is an improvement, and [eastwood](https://github.com/jonase/eastwood) or one of the linters we use on zebra agrees